test/e2e/block_storage_init_test.go

```diff
@@ -7,33 +7,29 @@
 package e2e
 
 import (
-	"testing"
-	"time"
-
 	"github.com/orbs-network/orbs-network-go/test"
 	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
 )
 
 func TestInitialBlockHeight(t *testing.T) {
-	const expectedBlocks = 500
 	if testing.Short() {
 		t.Skip("Skipping E2E tests in short mode")
 	}
 
-	runMultipleTimes(t, func(t *testing.T) {
-		h := NewAppHarness()
+	h := NewAppHarness()
+
+	if h.envSupportsLoadingWithCannedBlocksFile() {
```

Comment:
May be a bit cleaner to have a `h.expectedInitialBlockHeight` setting which is 500 when using the canned file and 0 otherwise, and have the test be identical for both flows.

Reply:
I don't want to hide away the fact that the test does nothing (in some cases). Prefer this limitation to be explicit